auth-web/src/components/auth/MaintainBusinessInfo.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-container id="maintain-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Manage and Maintain Your Business</h2>
+        <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Sub Bullet Points -->
+        <v-list-item class="list-item list-item-sub" v-for="(item, index) in subBulletPoints" :key="`Sub-index: ${index}`">
```

Comment:
This "sub list" works atm but it's not really a sub of the main list is it? It's a second, indented list. I don't know how to say this but it's bugging me. Either it should be a true sub-list of the main list, or it should be named something else, so that it doesn't confuse a future developer or require a refactor to make it work as a true sub list.

Reply:
hahah we can't have that! I will update this 👍

Reply:
Updated in next commit!